code/api/src/modules/user/tests/userQuery.test.js

```diff
@@ -197,4 +197,59 @@ describe('User queries', () => {
 
     expect(bad_request.body.data.user).to.eq(null);
   })
+
+  it('logs in a user', async () => {
+    const user_request = await request(server)
+      .post('/')
+      .send({ query: '{ userLogin(email: "user@crate.com", password: "123456", role: "USER") { user {id name email password role image address_line1 address_line2 city state zipcode description} token } }'});
```

Comment:
I'm glad that login resolver now exposes all this data for the FE. Thanks for ensuring that